Toggl.Core.Tests/UI/ViewModels/NotificationSettingsViewModelTests.cs

```diff
@@ -69,12 +70,12 @@ public async Task GetsInitialisedToTheProperValue(bool permissionGranted)
         public sealed class TheRequestAccessAction : NotificationSettingsViewModelTest
         {
             [Fact, LogIfTooSlow]
-            public async Task OpensAppSettings()
+            public async Task RequestThePermission()
```

Comment:
Can we rephrase it to this?
```suggestion
            public async Task RequestsPermission()
```